File: Pset's/pset6/mario/less/mario.py
# importing function from cs50 lib
from cs50 import get_int

# initializing height
height = 0

# prompting user to give us height in proper range
while height > 8 or height < 1:
    height = get_int("Height: ")

# Creating height
for row in range(height):
    # Creating width
    for space in range(height - row - 1):
        # add space (if needed)
        print(" ", end="")
    for column in range(row + 1):
        # add hash
        print("#", end="")
    # printing new line
    print()

File: Pset's/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, row, column, space;
    
    // Promt the user to give height
    do
    {
        height = get_int("What is the height of pyramid?: ");
    }
    // Integer input is restricted between 1 and 8
    while (height < 1 || height > 8);
    
    // Creating height
    for (row = 0; row < height; row++)
    {
        // Creating width
        for (space = 0; space < height - row - 1; space++)
        {
            // Putting spaces in pyramid first (if needed)
            printf(" ");
        }
        for (column = 0; column <= row; column++)
        {
            // Putting hashes in pyramid
            printf("#");
        }
        printf("\n");
    }
}